packages/in_app_purchase/lib/src/billing_client_wrappers/billing_client_wrapper.dart

```diff
@@ -73,6 +74,26 @@ class BillingClient {
     return channel.invokeMethod("BillingClient#endConnection()", null);
   }
 
+  /// Returns a list of [SkuDetailsWrapper]s that have [SkuDetailsWrapper.sku]
+  /// in [skusList], and [SkuDetailsWrapper.type] matching [skuType].
```

Comment:
s/[skusList]/\`skusList\`/
s/[skuType]/\`skuType\`/

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#dartdoc-specific-requirements